docs/faq.md

```diff
@@ -202,6 +202,60 @@ jib.from.image = 'custom-base-image'
 
 We currently support adding a custom directory with an **incubating** feature. This feature may change in later versions. If your application needs to use custom files, place them into the `src/main/jib` folder. Files placed here will be added to the filesystem of the container. For example, `src/main/jib/foo/bar` would add `/foo/bar` into the container filesystem.
 
+### <a id="i-want-more-control-over-which-files-to-place-in-the-extra-directory"></a>I want more control over which files to place in the extra directory/where they end up in the container
+
+If the current extra directory design doesn't meet your needs (e.g. the extra files you want are generated outside the extra directory, and you want a different directory structure in the container), you can use additional goals/tasks to move the files to the configured extra directory between compile time and package time.
+
+<details>
+<summary>File copying examples</summary>
+<p>
+
+#### Maven
+
+In Maven, you can use the `maven-resources-plugin` to copy files to your extra directory. In your `pom.xml`:
+
+```xml
+<plugins>
+  ...
+  <plugin>
+    <artifact>maven-resources-plugin</artifact>
+    <version>3.1.0</version>
```

Comment:
Not the most recent version. Any reason?

Reply:
It was what I saw [here](https://maven.apache.org/plugins/maven-resources-plugin/examples/copy-resources.html)

Reply:
OK, not a big deal. :)